_posts/fork/2020-3-25-Simple.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: theme
+title: Simple
+category: fork
```

Comment:
This is theme a "fork" of an existing theme? If not, the category should be "theme"